presto-native-execution/README.md

```diff
@@ -234,6 +247,16 @@ Run CLion:
   2. Run `Presto Client` Application (see above on how to create and setup the configuration) inside IntelliJ
 * You can start from `show tables;` and `describe table;` queries and execute more queries as needed.
 
+### Run Presto Coordinator + Worker + Sidecar
+* Note that everything below can be done without using IDEs by running command line commands (not in this readme).
+* Add a property `presto.default-namespace=native.default` to `presto-native-execution/etc/config.properties`.
+* Run `NativeSidecarPluginQueryRunner` from IntelliJ and wait until it starts (`======== SERVER STARTED ========` is displayed in the log output).
+* Scroll up the log output and find `Discovery URL http://127.0.0.1:50555`. The port is 'random' with every start.
+* Copy that port (or the whole URL) to the `discovery.uri` field in `presto/presto-native-execution/etc/config.properties` for the worker to discover the Coordinator and in `presto/presto-native-execution/etc_sidecar/config.properties` for the sidecar to discover the Coordinator.
```

Comment:
I don't follow why the side-car needs to discover the co-ordinator... Isn't sidecar just worker exposing some more end-points ?

Reply:
Maybe what is meant here is "... for the worker to announce itself to the Coordinator ..."